app/components/paperShow/components/collectionBox.scss

```diff
@@ -0,0 +1,375 @@
+@import "../../../variables";
+
+$color-mono-00: #1F2A36;
+$color-mono-01: #34495e;
+$color-mono-03: #77828C;
+$color-mono-04: #9AA7B5;
+$color-mono-06: #DAE1E8;
+$color-mono-07: #F0F1F2;
+$color-mono-08: #F7F9FA;
+$color-mono-09: #FAFBFC;
+$color-mono-10: #FDFEFF;
+
+$color-blue-01: #133F7A;
+$color-blue-03: #3e7fff;
+$color-blue-07: #E6EEFF;
+
+
+.fab {
+  position: fixed;
+  bottom: 24px;
+  right: 24px;
+  z-index: 100;
+
+  * {
+    margin: 0;
+    padding: 0;
+    font-family: "Roboto", sans-serif;
+    font-size: 14px;
+    line-height: 1.5;
+    color: $color-mono-00;
+    -webkit-font-smoothing: antialiased;        //anti-aliasing
+    text-shadow: 1px 1px 1px rgba(0,0,0,0.004); //anti-aliasing
+    outline: none;
+    ul {list-style: none;}
+  }
+
+  .paper_item_a {
+    padding: 12px 0;
+    &:first-child {
+      padding-top: 0;
+    }
+    &:not(:last-child) {
+      border-bottom: 1px solid $color-mono-06;
+    }
+    &:hover {
+      cursor: pointer;
+      .paper_item_a__paper__title {
+        color: $color-blue-03;
+      }
+    }
+    &__paper {
+      display: inline-block;
+      vertical-align: top;
+      width: 316px;
```

Comment:
I prefer to pull out this kind of value to a file scope or global scope as a variable.

Reply:
Do you mean pulling this to common component?